docs/advanced_topics/documents/custom_document_model.md

```diff
@@ -0,0 +1,67 @@
+(custom-document-model)=
+
+# Custom document model
+
+An alternate `Document` model can be used to add custom behaviour and
+additional fields.
+
+You need to complete the following steps in your project to do this:
+
+- Create a new document model that inherits from
+  `wagtail.documents.models.AbstractDocument`. This is where you would
+  add additional fields.
+- Point `WAGTAILDOCS_DOCUMENT_MODEL` to the new model.
+
+Here's an example:
+
+```python
+# models.py
+from wagtail.documents.models import Document, AbstractDocument
+
+class CustomDocument(AbstractDocument):
+    # Custom field example:
+    source = models.CharField(
+        max_length=255,
+        blank=True,
+        null=True
+    )
+
+    admin_form_fields = Document.admin_form_fields + (
+        # Add all custom fields names to make them appear in the form:
+        'source',
+    )
+```
+
+Then in your settings module:
+
+```python
+# Ensure that you replace app_label with the app you placed your custom
+# model in.
+WAGTAILDOCS_DOCUMENT_MODEL = 'app_label.CustomDocument'
+```
+
+```{topic} 
```

Comment:
why is this not shown in PR's docs [build](https://wagtail--8131.org.readthedocs.build/en/8131/advanced_topics/documents/custom_document_model.html) ?